src/pages/DetailPage/SideImg.js

```diff
@@ -0,0 +1,55 @@
+import styled from 'styled-components';
+
+const SideImg = props => {
+  console.log('prop', props.data.thumbnail__url);
+  return (
+    <Flexdiv>
+      <Ul>
+        <Imgli>
+          <Img
+            src="https://ojsfile.ohmynews.com/STD_IMG_FILE/2010/0809/IE001224096_STD.jpg"
+            alt="sideimg"
+          />
+        </Imgli>
+      </Ul>
+      <Mainimg>
+        <img src={props.data.thumbnail__url} alt="detailimg" />
+      </Mainimg>
+    </Flexdiv>
+  );
+};
+const Ul = styled.ul`
+  display: flex;
+  flex-direction: column;
```

Comment:
세로 정렬은 block 요소들의 기본 속성이기 때문에 flex + column 속성이 불필요해보입니다
```suggestion
const Ul = styled.ul`
```

Reply:
수정했습니다!